components/frontend/src/dynamo/frontend/main.py

```diff
@@ -176,6 +176,12 @@ def parse_args():
         default=None,
         help="Prefix for Dynamo frontend metrics. If unset, uses DYN_METRICS_PREFIX env var or 'dynamo_frontend'.",
     )
+    parser.add_argument(
+        "--tool-call-parser",
```

Comment:
Since the frontend can serve many models - should this be a frontend argument? Or backend/worker argument passed through ModelDeploymentCard/RuntimeConfig etc. that the frontend can load on demand when worker is discovered?

ex: `python -m dynamo.vllm` instead of `python -m dynamo.frontend`
```bash
python -m dynamo.vllm --tool-call-parser ...
```